script/build/linux

```diff
@@ -4,10 +4,15 @@ set -ex
 
 ./script/clean
 
-TAG="docker-compose"
-docker build -t "$TAG" .
-docker run \
-    --rm --entrypoint="script/build/linux-entrypoint" \
-    -v $(pwd)/dist:/code/dist \
-    -v $(pwd)/.git:/code/.git \
-    "$TAG"
+DOCKER_COMPOSE_GITSHA="$(script/build/write-git-sha)"
+TMP_CONTAINER="tmpcontainer-${DOCKER_COMPOSE_GITSHA}"
+TAG="docker/compose:tmp-glibc-linux-binary-${DOCKER_COMPOSE_GITSHA}"
+
+docker build -t "${TAG}" . \
+       --build-arg BUILD_PLATFORM=debian \
+       --build-arg GIT_COMMIT="${DOCKER_COMPOSE_GITSHA}"
+docker create --name "${TMP_CONTAINER}" "${TAG}"
```

Comment:
alternatively; don't give it a name, and capture the container ID (which is printed when creating the container)

Reply:
Done